index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <!-- header containing the logo and the tool bar-->
+    <nav id="tool-bar">
+        <div id="logo-placeholder">
+            <p id="welcome" class="flg">
+                Welcome
+            </p>
+        </div>
+        <div id="normal-buttons">
+            <div class="unions">
+                <img src="./images/Union.png" alt="union icon">
+            </div>
+        </div>
+    </nav>
+
+    <!-- Placing the background image in this block-->
+     <div id="background-image">
+        <img src="./images/header-illsutration-mobile.svg" alt="header illsutration mobile">
+     </div>
+
+     <!-- The main heading information on the page-->
+    <main>
+            <h1 id="primary" class="flg">
+                Hey there. I’m Mariane <br><span>I’m a software developer</span>
+            </h1>
+            <p id="support" class="flg">
+                I can help you build a product , feature or website Look through some of my work 
+                and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <div id="social" class="flg">
+                <img src="./images/smi.svg" alt="social media icons">
+            </div>
+            <div id="action" class="flg">
+                <div id="action-button">
+                    <img src="./images/Union.svg" alt="action button">
+                </div>
+            </div>
```

Comment:
Great job @arabiu033 🤝

- [ ] Kindly surround each of these images with `a` tags or `button` tags. This is because they are clickable links.